### PR TITLE
Add test cases for JIGSAW IGNORE css pattern

### PR DESCRIPTION
**Pull request by piddubnyi**
Introduced test cases to test the path when the pattern matched and not matched.
Solving https://github.com/jcabi/jcabi-w3c/issues/32

### CODE REVIEW

**Comment by dmarkov** [comment]
@piddubnyi Let me find a reviewer for this pull request, thanks for submitting it

**Comment by dmarkov** [comment]
@caarlos0 could you please review this PR

File: src/test/java/com/jcabi/w3c/DefaultCssValidatorTest.java [hunk 1]
```diff
@@ -57,15 +57,7 @@
...
-                StringUtils.join(
-                    "<env:Envelope",
-                    " xmlns:env='http://www.w3.org/2003/05/soap-envelope'>",
-                    "<env:Body><m:cssvalidationresponse",
-                    " xmlns:m='http://www.w3.org/2005/07/css-validator'>",
-                    "<m:validity>true</m:validity>",
-                    "<m:checkedby>W3C</m:checkedby>",
-                    "</m:cssvalidationresponse></env:Body></env:Envelope>"
-                )
+                this.validResponce()
```
Comment (caarlos0) [inline src/test/java/com/jcabi/w3c/DefaultCssValidatorTest.java]: @piddubnyi typo in repon **s** e

**Comment by caarlos0** [comment]
@piddubnyi one minor comment above

**Comment by piddubnyi** [comment]
@caarlos0 Thanks, fixed.

**Comment by caarlos0** [comment]
@rultor merge

**Comment by caarlos0** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@caarlos0 I just added **19 mins** to your account, many thanks for your contribution (56a747222416275f6f00014e).. 116 hours and 28 mins spent here. you're getting extra minutes here (c=4). added +19 to your rating, now it is equal to [+466](http://www.netbout.com/b/35095?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**